internal/cli/apps.go

```diff
@@ -179,6 +186,7 @@ func useAppCmd(cli *cli) *cobra.Command {
 }
 
 func listAppsCmd(cli *cli) *cobra.Command {
+	var inputs struct{ Reveal bool }
```

Comment:
```suggestion
	var inputs struct{
		Reveal bool
	}
```

While it's more lines, consistency is king.